templates/d1/src/db/schema.ts

```diff
@@ -0,0 +1,10 @@
+import {sql} from "drizzle-orm";
```

Comment:
nit: could you run a formatter `prettier`/`biome` so that code style gets normalized? Note the spacing between brackets, and the intermittent use of double `"` and single `'` apostrophes